src/lib/y2partitioner/dialogs/fstab_options.rb

```diff
@@ -18,42 +19,13 @@ def title
         _("Fstab Options:")
       end
 
-      def wizard_create_dialog(&block)
-        Yast::UI.OpenDialog(layout)
-        block.call
-      ensure
-        Yast::UI.CloseDialog()
-      end
-
-      def should_open_dialog?
-        true
-      end
-
       def contents
-        HVSquash(Widgets::FstabOptions.new(@options))
-      end
-
-      def layout
-        VBox(
-          HSpacing(50),
-          # heading text
-          Left(Heading(Id(:title), title)),
-          VStretch(),
-          VSpacing(1),
-          HBox(
-            HStretch(),
-            HSpacing(1),
-            ReplacePoint(Id(:contents), Empty()),
-            HStretch(),
-            HSpacing(1)
-          ),
-          VSpacing(1),
-          VStretch(),
-          ButtonBox(
-            PushButton(Id(:help), Opt(:helpButton), Yast::Label.HelpButton),
-            PushButton(Id(:ok), Opt(:default), Yast::Label.OKButton),
-            PushButton(Id(:cancel), Yast::Label.CancelButton)
-          )
+        HBox(
+          HStretch(),
+          HSpacing(1),
+          HVSquash(Widgets::FstabOptions.new(@options)),
+          HStretch(),
```

Comment:
I am not sure if it make sense to have in HBox two Stretch, as first stretch should already eat all space, if I am not wrong.